JavaScript/JavaScriptSDK/Telemetry/Common/DataSanitizer.ts

```diff
@@ -156,6 +166,23 @@ module Microsoft.ApplicationInsights.Telemetry.Common {
             return measurements;
         }
 
+        public static sanitizeId(id: string) {
```

Comment:
you changed the other method to take a value for maxlength that has a default, why not do the same here?  (or why is there a different method for ids at all, when it's the same rules but different trim length?

Reply:
Combined sanitizeUrl and sanitizeId() to call single sanitizeInput() method.